src/pages/Scholarships.jsx

```diff
@@ -11,7 +11,9 @@ import Button from '@material-ui/core/Button';
 import Scholarships from '../models/Scholarships';
 import ScholarshipList from '../components/ScholarshipList';
 import FilterBar from '../components/FilterBar';
-import qParams from '../lib/QueryParams';
+import { qParams, sortParams } from '../lib/QueryParams';
+
+const DEFAULT_SORT_BY = 'deadlineAsc';
```

Comment:
Putting `sortParams`/`sortOptions` next to this line (maybe before) is probably an ideal place since they're related.

Reply:
So removing it from the import and just placing it on this current file? The sortParams but in another comment you mentioned to change the name to sortOptions right. Ill get this change in